src/main/java/org/example/hospital/controler/DepartmentController.java

```diff
@@ -31,22 +33,24 @@ public String removeDepartment(@PathVariable Long departmentId){
     }
```

Comment:
Maybe it is better to customize the HTTP response status and message instead of returning a string.